docs/src/_layout.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, user-scalable=no">
     <title>{{title if title}}{{" | " + titleSegments | join(" |") if titleSegments}}{{" | " if title}}@esri/hub.js</title>
     {% if description %}
-    <meta name="description" content="This is an example of a meta description. This will often show up in search results.">
+    <meta name="description" content="A modular, high quality toolkit for working with <a href='https://hub.arcgis.com/'>ArcGIS Hub</a>">
```

Comment:
can meta tags contain anchor tags?

Reply:
> The value must be a free-form string that describes the page ... There must not be more than one `meta` element where the `name` attribute value is an ASCII case-insensitive match for `description` per document.

https://html.spec.whatwg.org/multipage/semantics.html#meta-description

i'm pretty sure that means "no" 🤔